### ISSUE TITLE
Error: implicit conversion increases floating-point precision on Xcode 7.3 when archiving

### ISSUE THREAD

**Issue by brendand** [abbeycode/UnzipKit#26]
When you archive for OS X or iOS, Xcode 7.3 reports the following error:

```
/Users/brendan/Projects/iPhone/UnzipKit/Source/UZKArchive.m:404:58: error: implicit conversion increases floating-point precision: 'float' to 'CGFloat' (aka 'double') [-Werror,-Wdouble-promotion]
                        progress(info, bytesDecompressed / totalSize.floatValue);
                        ~~~~~~~~       ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
```

Changing that line to totalSize.doubleValue fixes the problem.

**Comment by abbeycode** [comment]
Thanks, I'll take a look once I upgrade to El Capitan.

**Comment by abbeycode** [comment]
This has been addressed in `master` and will be included in the 1.7 release

**Comment by abbeycode** [comment]
v1.7 is live on CocoaPods now, including this fix. Thanks again @brendand for pointing this out!
